packages/strapi-plugin-content-manager/admin/src/utils/getAttributeInfos.js

```diff
@@ -0,0 +1,6 @@
+import { get } from 'lodash';
+
+const getType = (schema, attrName) => get(schema, ['attributes', attrName, 'type'], '');
+const getOtherInfos = (schema, arr) => get(schema, ['attributes', ...arr], '');
+
+export { getType, getOtherInfos };
```

Comment:
do we really need helpers for a `get(..)` ?

Reply:
Yes so I don't have to repeat the same path.